Week3/homework/step2-6.js

```diff
@@ -4,13 +4,21 @@ const arr2d = [[1, 2], [3, 4], [5, 6]];
 const arr3d = [[[1, 2], [3, 4]], [[5, 6], [7, 8]]];
 
 function printArray2d(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+  let result = '';
+  for (let i = 0; i < arr.length; i++) {
+    result += arr[i].join('');
+  }
+  return console.log(result);
 }
```

Comment:
Perhaps you peeked at the unit test to see what it expected, however, you could just console.log each array element directly:

```js
function printArray2d(arr) {
  for (let i = 0; i < arr.length; i++) {
    for (let j = 0; j < arr[i].length; j++) {
      console.log(arr[i][j]);
    }
  }
}
```

Having said that, your solution passes the test.